src/DependencyInjection/Compiler/AddTemplatesCompilerPass.php

```diff
@@ -26,7 +30,7 @@ class AddTemplatesCompilerPass implements CompilerPassInterface
 {
     public function process(ContainerBuilder $container)
     {
-        $overwrite = $container->getParameter('sonata.admin.configuration.admin_services');
+        // NEXT_MAJOR: Remove this line.
```

Comment:
if we remove this line there is no template vars, all the method is to be removed?